Managers/FrameMan.cpp

```diff
@@ -120,21 +121,27 @@ namespace RTE {
 #ifdef _WIN32
 		if (m_ForceOverlayedWindowGfxDriver) {
 			m_GfxDriver = GFX_DIRECTX_OVL;
-			g_ConsoleMan.PrintString("SYSTEM: Using overlay DirectX windowed driver!");
+			m_GfxDriverMessage = "SYSTEM: Using overlay DirectX windowed driver!";
 		} else if (m_ForceNonOverlayedWindowGfxDriver) {
 			m_GfxDriver = GFX_DIRECTX_WIN;
-			g_ConsoleMan.PrintString("SYSTEM: Using non-overlay DirectX windowed driver!");
+			m_GfxDriverMessage = "SYSTEM: Using non-overlay DirectX windowed driver!";
 		} else if (m_ForceVirtualFullScreenGfxDriver) {
 			m_GfxDriver = GFX_DIRECTX_WIN_BORDERLESS;
-			g_ConsoleMan.PrintString("SYSTEM: Using DirectX fullscreen-windowed driver!");
+			m_GfxDriverMessage = "SYSTEM: Using DirectX fullscreen-windowed driver!";
 		} else {
 			m_GfxDriver = GFX_AUTODETECT_WINDOWED;
 		}
 #else
-		m_GfxDriver= (m_ResX * m_ResMultiplier == m_ScreenResX && m_ResY * m_ResMultiplier == m_ScreenResY) ? GFX_AUTODETECT_FULLSCREEN : GFX_AUTODETECT_WINDOWED;
+		m_GfxDriver = (m_ResX * m_ResMultiplier == m_ScreenResX && m_ResY * m_ResMultiplier == m_ScreenResY) ? GFX_AUTODETECT_FULLSCREEN : GFX_AUTODETECT_WINDOWED;
 #endif
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void FrameMan::PrintSelectedGfxDriverMessage() const {
```

Comment:
Do we normally get a driver message or is this a rare occurrence? I'm wondering if this should be renamed slightly to better describe it, something like `PrintGfxDriverMessageIfItExists()` or whatever; basically saying that there won't necessarily be a message to print. I feel like `Selected` doesn't imply that very clearly.